include/utility.hpp

```diff
@@ -412,7 +400,7 @@ namespace giml {
         void removeAt(size_t indexToRemove) {
             if (indexToRemove >= this->length || indexToRemove < 0) {
                 printf("Array access out of bounds");
-                throw std::out_of_range("Index out of range");
+                //throw std::out_of_range("Index out of range");
```

Comment:
Quick fix for now needs to be "fixed" before we merge to `main`. I know this is mine but I am being equally critical. We need to figure out error handling, maybe a sort of flag system?

Reply:
`<stdexcept>` undermines Gimmel's portability promise. I'd prefer to handle erroneous values with clamping, wrapping, etc "fix it for them" instead of throwing a fit. I.e. how `CircBuff` simply clamps the delay val based on size

Reply:
We will meet to discuss+weigh our options on error handling but that's a good point about portability. I feel we should give error control to our users though, instead of masking it. There should be a way to flag it but still have the program run, these are not mutually exclusive.